pkg/network/status/status.go

```diff
@@ -105,10 +111,10 @@ type Manager interface {
 type Prober struct {
 	logger *zap.SugaredLogger
 
-	// mu guards ingressStates and podStates
+	// mu guards ingressStates and podContexts
 	mu            sync.Mutex
 	ingressStates map[string]*ingressState
-	podStates     map[string]*podState
+	podContexts   map[string]*cancelContext
```

Comment:
I'd suggest just `cancelContext`.

Reply:
Are you suggesting:
1) renaming `podContexts` to `cancelContexts` ?
2) replacing `*cancelContext` by `cancelContext` ?

Reply:
2.

Reply:
Done.